applications/RANSApplication/custom_strategies/relaxed_dof_updater.h

```diff
@@ -0,0 +1,202 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_RELAXED_DOF_UPDATER_H_INCLUDED )
+#define  KRATOS_RELAXED_DOF_UPDATER_H_INCLUDED
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+
+#ifdef KRATOS_USING_MPI // mpi-parallel compilation
+class Epetra_Import;
+#endif
+
+
+
+namespace Kratos
+{
+///@addtogroup KratosCore
+///@{
+
+///@name Kratos Classes
+///@{
+
+/// Utility class to update the values of degree of freedom (Dof) variables after solving the system.
+/** This class encapsulates the operation of updating nodal degrees of freedom after a system solution.
+ *  In pseudo-code, the operation to be performed is
+ *  for each dof: dof.variable += dx[dof.equation_id]
+ *  This operation is a simple loop in shared memory, but requires additional infrastructure in MPI,
+ *  to obtain out-of-process update data. RelaxedDofUpdater takes care of both the operation and the eventual
+ *  auxiliary infrastructure.
+  */
+template< class TSparseSpace >
+class KRATOS_API(RANS_APPLICATION) RelaxedDofUpdater
```

Comment:
Shouldn't this derive from the common `DofUpdater`?

Reply:
done :)